center/src/main/java/com/microsoft/hydralab/center/config/SecurityConfiguration.java

```diff
@@ -19,7 +19,7 @@
 @Configuration
 @EnableWebSecurity
 @EnableGlobalMethodSecurity(prePostEnabled = true)
-@Profile({"default", "dev", "release"})
+@Profile({"default", "dev", "release", "docker"})
```

Comment:
Could we remove this line?

Reply:
Without enabling this, InMemoryUserDetailsManager would be re-enabled, and some other issues would occur such as websocket cannot connect.